.releaserc.cjs

```diff
@@ -77,14 +77,236 @@ const Runtimes = {
+            { type: "feat", section: "Features" },
+            { type: "fix", section: "Fixes" },
+            { type: "chore", section: "Maintenance" },
+            { type: "docs", section: "Maintenance" },
+            { type: "revert", section: "Fixes" },
+            { type: "style", section: "Miscellaneous" },
+            { type: "refactor", section: "Miscellaneous"},
+            { type: "perf", section: "Miscellaneous" },
+            { type: "test", section: "Miscellaneous" },
+          ],
+        },
+        releaseRules: [
+          { type: "docs", release: "patch" },
+          { type: "revert", release: "patch" },
+          { type: "chore", release: "patch" },
+        ],
+      },
+    ],
     // Based on the commits generate release notes
-    "@semantic-release/release-notes-generator",
+    [
+      "@semantic-release/release-notes-generator",
+      {
+        preset: "conventionalcommits",
+        presetConfig: {
+          types: [
+            {
+              type: "feat",
+              scope: "dafny",
+              section: "Features -- All Languages",
+              hidden: false,
+            },
+            {
+              type: "feat",
+              scope: "java",
+              section: "Features -- Java",
+              hidden: false,
+            },
+            {
+              type: "feat",
+              scope: "dotnet",
+              section: "Features -- DotNet",
+              hidden: false,
+            },
+            {
+              type: "feat",
+              scope: "python",
+              section: "Features -- Python",
+              hidden: false,
+            },
+            {
+              type: "feat",
+              scope: "go",
+              section: "Features -- Go",
+              hidden: false,
+            },
+            {
+              type: "feat",
+              scope: "rust",
+              section: "Features -- Rust",
+              hidden: false,
+            },
+
+            {
+              type: "fix",
+              scope: "dafny",
+              section: "Fixes -- Dafny",
+              hidden: false,
+            },
+            {
+              type: "fix",
+              scope: "java",
+              section: "Fixes -- Java",
+              hidden: false,
+            },
+            {
+              type: "fix",
+              scope: "dotnet",
+              section: "Fixes -- DotNet",
+              hidden: false,
+            },
+            {
+              type: "fix",
+              scope: "python",
+              section: "Fixes -- Python",
+              hidden: false,
+            },
+            { type: "fix", scope: "go", section: "Fixes -- Go", hidden: false },
+            {
+              type: "fix",
+              scope: "rust",
+              section: "Fixes -- Rust",
+              hidden: false,
+            },
+
+            {
+              type: "chore",
+              scope: "dafny",
+              section: "Maintenance -- Dafny",
```

Comment:
Should this also be "All Languages"? (I could go either way, sometimes it really isn't for all languages). Ditto for other Dafny scopes

Reply:
makes sense